notebook.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    print ('you know {} ways to say hello world'.format(counter))
</pre><p>Wow cool syntax here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meg-delaunay/reviewnb-notebook-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>